INSTALL.md

```diff
@@ -0,0 +1,163 @@
+# Use Helm to quickly deploy Dragonfly+Nydus on K8S
+# Introduction
+In the cloud-native scenario, the operation of business systems is inseparable from the creation, distribution, and operation of images. In the process of using the current image, the full image needs to be pulled from the warehouse to the local, and then decompressed and mounted by the container engine before it can be provided to the container. In the actual production process, the distribution and loading of a large number of images is often the bottleneck that limits the speed of container startup.
+
+In order to solve the above problems, we introduced the image acceleration solution of Nydus+Dragonfly. However, in the past deployment process, users need to manually install many components, which is costly. To this end, this article will introduce how to use the Kubernetes package management tool Helm to deploy the Dragonfly+Nydus solution on the K8S cluster simply and quickly.
+# 1. Prepare k8s cluster
+## 1.1 Use kind to create a cluster locally
+In addition to kind, you can also deploy K8s clusters in other ways, but further configuration adjustments may be required.
+In the remark chapter, we gave an example of deployment on [Alibaba Cloud ACK](https://www.alibabacloud.com/product/kubernetes)
+### (1) Install Kind
+```
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64
+chmod +x ./kind
+sudo mv ./kind /usr/local/bin/kind
+```
+### (2) Download config and create cluster
+```
+wget https://raw.githubusercontent.com/dragonflyoss/Dragonfly2/main/test/testdata/containerd/config.toml
+
+cat <<EOF > kind-config.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: dual
+nodes:
+  - role: control-plane
+    image: kindest/node:v1.23.17
+    extraPortMappings:
+      - containerPort: 65001
+        hostPort: 65001
+        protocol: TCP
+    extraMounts:
+      - hostPath: ./config.toml
+        containerPath: /etc/containerd/config.toml
+      - hostPath: /tmp/artifact
+        containerPath: /tmp/artifact
+      - hostPath: /dev/fuse
+        containerPath: /dev/fuse
+EOF
+
+kind create cluster --config ./kind-config.yaml --name kind --wait 300s
+```
+It may take a long time to wait here. You can use docker to pre-pull the image to solve the problem:
+
+`docker pull kindest/node:v1.23.17`
+
+![image.png](./docs/img/1680576955281-5cf9d091-0cba-422c-9e46-553fadab86fc.png)
+## 1.2 Install kubectl
```

Comment:
You can remove `Install kubectl`、`Install Helm` and `Install Kind`. Replace with `Prerequisites `, refer to https://d7y.io/blog#prerequisites.

Reply:
Fixed.